### ISSUE TITLE
Theme accessibility-ready guidelines: address CSS generated content alternative text

### ISSUE THREAD

**Issue by joedolson** [wpaccessibility/wp-a11y-docs#362]
- [x] I've checked the [existing issues for this change](https://github.com/wpaccessibility/wp-a11y-docs/issues)

**URL**
https://wpaccessibility.org/docs/topics/theme-guidelines/alternative-text/

**Title**
Alternative text on images and graphics

**What Needs Editing?**

Currently, we address several specific resolutions to decorative images needing to be hidden. We do not address the option to use CSS generated content alternatives, e.g. `content: '\f26e' / '';`. This means that to strictly meet the guidelines, a theme author using generated content would potentially have to change their output HTML in order to meet the requirement.

However, particularly for themes that have already been released, making HTML changes can introduce breaking changes for child themes and extenders, which may make them unwilling to make those changes.

I think we should allow defining empty alternative text for CSS generated content as an option.

See https://wordpress.slack.com/archives/CEKKABGJD/p1782992781431179

**Comment by rianrietveld** [comment]
@joedolson This could also be a topic page. Content in CSS.
